pp_hot.c

```diff
@@ -3896,6 +3896,7 @@ PP(pp_iter)
     itersvp = CxITERVAR(cx);
     assert(itersvp);
 
+    {
     switch (CxTYPE(cx)) {
 
     case CXt_LOOP_LAZYSV: /* string increment */
```

Comment:
This commit doesn't do what it says it does.  It adds the block around the switch (which isn't purely a whitespace change), and doesn't actually indent the block.

Later commits add lines indented further, so the HEAD includes:
```
        STRLEN maxlen = 0;
        const char *max = SvPV_const(end, maxlen);
            bool pad_it = FALSE;
        if (DO_UTF8(end) && IN_UNI_8_BIT)
```

I suspect a `git rebase --ignore-whitespace` bit you here.

Reply:
It did. Fixed in the rebase, and thanks for correctly diagnosing the problem.